networking/src/NTP/Client.hs

```diff
@@ -125,6 +124,14 @@ handleCollectedResponses cli = do
   where
     handleE = logError . sformat ("ntpMeanSelection: "%shown)
 
+gatheredAllResponses :: NtpClient m -> STM Bool
+gatheredAllResponses cli = do
```

Comment:
`cli` is a bit confusing as a variable name (it reads in my mind as "command line interface"). What about `ntpClient`, plain and simple?

Reply:
Can't say it's simple, taking into account that this thing is used all over the module 😒